analyzers/tests/SonarAnalyzer.UnitTest/TestCases/LoopsAndLinq.cs

```diff
@@ -0,0 +1,228 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Tests.Diagnostics
+{
+    public class S3267
```

Comment:
what about the scenarios with LINQ statements inside

```
foreach (var element in collection.Select(...).Where(...))
{
  if (condition(element))
  {

  }
}

```

Reply:
We will still raise. I will add a test case.